mllib/src/main/scala/org/apache/spark/ml/clustering/BisectingKMeans.scala

```diff
@@ -117,7 +117,8 @@ class BisectingKMeansModel private[ml] (
     validateAndTransformSchema(schema)
   }
 
-  private[clustering] def predict(features: Vector): Int = parentModel.predict(features)
+  @Since("2.4.0")
```

Comment:
This would have to be 3.0.0.

I don't see a good reason not to expose this. CCing also maybe @mgaido91 ?

Reply:
I don't see a good reason either. I think it is fine to expose it.